src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/mappers/ContactDetailsMapper.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers;
+
+import uk.gov.hmcts.reform.bsp.common.model.shared.in.OcrDataField;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.bsp.common.mapper.AddressMapper.applyMappings;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.helper.CommonConditions.isApplicantRepresented;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.helper.CommonConditions.isRespondentRepresented;
+
+public class ContactDetailsMapper {
+
+    public static class CcdFields {
+        public static final String APPLICANT = CCDConfigConstant.APPLICANT_ADDRESS;
+        public static final String APPLICANT_SOLICITOR = CCDConfigConstant.SOLICITOR_ADDRESS;
+        public static final String APPLICANT_PHONE = CCDConfigConstant.APPLICANT_PHONE;
+        public static final String APPLICANT_EMAIL = CCDConfigConstant.APPLICANT_EMAIL;
+        public static final String APPLICANT_SOLICITOR_EMAIL = CCDConfigConstant.SOLICITOR_EMAIL;
+        public static final String APPLICANT_SOLICITOR_PHONE = CCDConfigConstant.SOLICITOR_PHONE;
+
+        public static final String RESPONDENT = CCDConfigConstant.RESPONDENT_ADDRESS;
+        public static final String RESPONDENT_SOLICITOR = CCDConfigConstant.RESP_SOLICITOR_ADDRESS;
+    }
+
+    /**
+     * We only store contact details to citizens or theirs solicitors (if they are represented). Never both.
+     *
+     * <p>We don't need to stored data about applicant's/respondent's contact details when they are represented.
```

Comment:
You can remove the <p>

Reply:
https://github.com/hmcts/finrem-case-orchestration-service/pull/293